test-network/scripts/deployCC.sh

```diff
@@ -284,13 +335,13 @@ commitChaincodeDefinition 1 2
 queryCommitted 1
 queryCommitted 2
 
-## Invoke the chaincode
-chaincodeInvokeInit 1 2
-
-sleep 10
-
-# Query chaincode on peer0.org1
-echo "Querying chaincode on peer0.org1..."
-chaincodeQuery 1
+## Invoke the chaincode - this does require that the chaincode have the 'initLedger'
```

Comment:
This switch conflates the --init-required flag with the idea of having a ledger initialization function, when the two are distinct. However, providing one flow is a demonstration of how it works, and having both of them as flags could make things even more confusing.

Reply:
Yes that was the idea to combine the two , if you use one you have to do the other and keep it simple to use.  The user is free to run their own init function that is not tied to the lifecycle initialization tracking process by doing an invoke with the application.